docs/usage/trad-cluster.md

```diff
@@ -0,0 +1,18 @@
+# Traditional Clusters
+
+Traditional clusters, or simply "clusters" in this document. Are machines that you can provision and run yourself.
```

Comment:
Again not sure the definition of a traditional cluster is appropriate. Maybe something like: Traditional clusters typically involve physical hardware that dedicated to running simulations within the host operating system.